src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -76,6 +76,10 @@ object ZincRunnerWorkerConfig {
  */
 object ZincRunner extends WorkerMain[ZincRunnerWorkerConfig] {
 
+  // Interrupting concurrent Zinc/Scala compilation with a shared ScalaInstance (and thus shared classloaders)
```

Comment:
It may be worth elaborating here on your hypothesis for why these errors are happening.